src/lib/tabs/tab-group.ts

```diff
@@ -224,6 +226,13 @@ export class MatTabGroup extends _MatTabGroupMixinBase implements AfterContentIn
     this._tabLabelSubscription.unsubscribe();
   }
 
+  /** Re-aligns the ink bar to the selected tab element. */
+  realignInkBar() {
```

Comment:
Can we add a test in for this?

Reply:
We could, although these tests that have to do with element dimensions/positioning tend to be very flaky.

Reply:
I am not able to make this work. I am subscribing to a change in the menu toggle (when the matsidenav is opening or closing), and then I realign the nav bar each time a toggle happens, but the aligning is still not correct :/

Reply:
I experimented using a timeout like this:
```
setTimeout(() => {
  this.tabGroup.realignInkBar();
}, 300);
```
This will align the inkbar, but it has to wait for the first animation to finish (being pushed because the sidenav is expanding), and then animate back to the correct position. I will try to make a Stackblitz to illustrate what I mean. Maybe I am doing something fundamentally wrong to begin with.